layer/logged_data/log_data_runner.py

```diff
@@ -114,8 +114,11 @@ def log(  # pylint: disable=too-many-statements
             elif LogDataRunner._is_tabular_dict(value):
                 if TYPE_CHECKING:
                     assert isinstance(value, dict)
-                dataframe = LogDataRunner._convert_dict_to_dataframe(value)
-                self._log_dataframe(value=dataframe, **kwargs)
+                # expand the dict into a group of logs
+                for item_key, item_value in value.items():
+                    self.log(
+                        data={item_key: item_value}, category=category, group_tag=tag
+                    )
```

Comment:
@mecevit this PR reverts the behavior from https://github.com/layerai/sdk/pull/92 -- are you cool with that?

Reply:
There is no change on the UI or the SDK side, it's just backend. So I'm fine

Reply:
No, I think there's a change in _both_ UI and SDK side.